research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+---
+menu: Proposals
+name: Tabs (Editor's Draft)
+path: /components/tabs
+pathToResearch: /components/tabs.research
+showInMenu: false
+---
+
+# Tabs Component Specification
+
+## Overview <a href="#overview" id="overview"></a>
+
+*Tabs* are a set of layered sections of content that display one panel of content at a time. Each tab panel has an associated tab element, that when activated, displays the panel. The list of tab elements is arranged along one edge of the currently displayed panel.
+
+### Background <a href="#background" id="background"></a>
+
+_Relevant historical or background information, related existing issues, etc._
+
+### Use Cases <a href="#use-cases" id="use-cases"></a>
+
+- Steve is researching for an upcoming vacation for him and his family. He opens his web browser and opens a new tab to make a rental car reservation and another for his flight information. Steve can quickly switch back and fourth between tabs to get the information he needs for the rental car reservation.
+
+- Monika visits a website to learn how to fix her leaky sink. The website displays a vertical tab interface with each step. Monika can jump ahead steps or go back to previous steps by selecting various tabs.
+
+### Non-goals <a href="#non-goals" id="non-goals"></a>
+
+_A list of use cases, features, or functionality which are **not** goals for the component._
+
+### Features <a href="#features" id="features"></a>
+
+- **Orientation:** Allows the tab list to be oriented horizontally above the tab content or vertically to the left or right (depending on language region) of the tab content.
+- **Supplemental content:** Offers a way to add content via start and end to the left and/or right (depending on language region) of the the tab list.
+- **ActiveIndicator:** Offers a way to add or remove an active indicator that highlights the currently active tab and animates to the next active tab. Defaults to true.
+- **ActiveTab:** Provides a reference to the currently active tab vis the `change` event.
+- **ActiveId:** Provides a way to set the active tab.
```

Comment:
In discussion with some developers recently it was suggested that this was less than pleasing in practice actually, and this matches my own experience from implementing and using this as elements many times.  Things that do this by ref attribute are more pleasing is the alternative feedback we've gotten - in other words, mark the tab's related 'element' as active with a boolean attribute or something or a method that takes the tab to make active. /me shrugs

Reply:
Sounds like this deserves an issue all of its own.

Reply:
most of them probably do, as I said they are mostly notes for me to remember to come back once merged

Reply:
https://github.com/WICG/open-ui/issues/191